.github/workflows/deploy.yml

```diff
@@ -32,4 +32,4 @@ jobs:
       - name: Build with Maven
         env:
           FTP_URL: ${{ secrets.LIVE_SERVER_FTP_URL }}
-        run: mvn -B -Dmaven.source.skip=true -Dmaven.test.skip=${{ github.event.inputs.skip-tests }} -pl :commons,:${{ github.event.inputs.server }} package -Dftp.url=$FTP_URL
\ No newline at end of file
+        run: mvn -B -Dmaven.source.skip=true -Dmaven.test.skip=${{ github.event.inputs.skip-tests }} -pl :${{ github.event.inputs.server }} -am package -Dftp.url=$FTP_URL
```

Comment:
note this change, removing explicit `:commons` compilation in favour of `-am` (also-make) which builds the parent and commons.